plugins/a11y-settings/msd-a11y-settings-manager.c

```diff
@@ -54,7 +54,7 @@ G_DEFINE_TYPE_WITH_PRIVATE (MsdA11ySettingsManager, msd_a11y_settings_manager, G
 static gpointer manager_object = NULL;
 
 static void
-apps_settings_changed (GSettings              *settings,
+apps_settings_changed (GSettings              *settings G_GNUC_UNUSED,
 		       const char             *key,
 		       MsdA11ySettingsManager *manager)
```

Comment:
indent?

Reply:
What indent? He are fixing warning... Re-indenting would add unneeded noise.

Reply:
Maybe in your repos.
Common behaviour in MATE repos is to fix when it can be done easily.

Reply:
there is no indentation problem here, and in the next comment the same answer: **no identation problems**

![2020-08-15_17-03](https://user-images.githubusercontent.com/7734191/90315160-7cd91780-df19-11ea-9b13-9d61ce251aaa.png)

Reply:
@sc0w Why did you use tabs in some function headers if you know you should use only spaces for the newly written code? You fixed the indent on some headers but there are a few function headers that you didn't fix.